src/main/java/appeng/crafting/CraftingTreeNode.java

```diff
@@ -20,9 +20,13 @@
 
 
 import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Collections'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)